src/shapes/Text/Text.ts

```diff
@@ -295,6 +296,16 @@ export class FabricText<
    */
   declare path?: Path;
 
+  /**
+   * The text decoration tickness for underline, overline and strikethrough
+   * The tickness is expressed in thousandths of fontSize ( em ).
+   * The original value was 1/15 that translates to 66.6667 thousandths.
+   * The choice of unit of measure is to align with charSpacing.
+   * @type Number
+   * @default
+   */
+  declare textDecorationTickness: number;
```

Comment:
is there no default value assigned?

Reply:
yes but in the default object.
Fabric5 used to do this to define default values for properties:
```js
Fabric.Text.prototype.fontSize = 40;
```
And if the developer wanted to change the default fontSize could do the same.

When moving to classes we realized that wasn't possible anymore.

if you do:
```js
class Text {
  fontSize = 40;
}
```
That is not going to be in the prototype but on the instance and there is no way to change it globally.

So we declare the configurable properties would be only declared and assigned from an object in the constructor.
Modifying that object is the only way to change the default value for all the new instances.